samples/inputs/mask-input/value-modes/index.html

```diff
@@ -0,0 +1,37 @@
+<html>
+  <head>
+    <title>Mask Input Value Modes</title>
+    <meta charset="UTF-8" />
+
+    <link rel="shortcut icon" href="https://static.infragistics.com/xplatform/images/browsers/wc.png" >
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Kanit&display=swap" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Titillium Web" />
+    <link rel="stylesheet" href="https://static.infragistics.com/xplatform/css/samples/shared.v5.css" type="text/css" />
+  </head>
+
+  <body>
+    <div id="root">
+      <div class="container sample center">
+        <igc-mask-input id="mask-input">
+          <igc-icon name="github" slot="prefix"></igc-icon>
+          <igc-icon name="github" slot="suffix"></igc-icon>
+          <span slot="helper-text">This is some helper text</span>
```

Comment:
Change prefix icon to something `text-related` like this https://www.svgrepo.com/show/4731/text-file.svg . Remove suffix.